src/Connection/Internal/Http1Parser.php

```diff
@@ -116,9 +116,9 @@ public function parse(string $data = null): ?Response
         if (!$this->bodyStarted && \in_array($this->state, [self::BODY_CHUNKS, self::BODY_IDENTITY, self::BODY_IDENTITY_EOF], true)) {
             $this->bodyStarted = true;
 
+            $eventInvoker = events();
             \assert($this->response !== null);
-
-            events()->responseBodyStart($this->request, $this->stream, $this->response);
+            $eventInvoker->responseBodyStart($this->request, $this->stream, $this->response);
```

Comment:
Why this change?

Reply:
The Psalm setting `rememberPropertyAssignmentsAfterCall="false"` means calling `events` loses the assertion that `$this->response !== null`

Reply:
Shouldn't we store `$this->response` in a variable then instead of the event invoker?

Reply:
Either way, but that does match the style elsewhere better. Updated.